tests/Unit/Core/Addon/Theme/ThemeTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Unit\Core\Addon\Theme;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Addon\Theme\Theme;
+
+class ThemeTest extends TestCase
+{
+    /**
+     * @var Theme
+     */
+    protected $theme;
+
+    public function testGetThemeAttributes(): void
+    {
+        $theme = new Theme(
+            [
+                'name' => 'foo',
+                'bar' => 'preston',
+                'directory' => 'a/',
+            ],
+            '', '');
```

Comment:
```suggestion
            '',
            ''
        );
```

Reply:
One argument per line is useful when each argument needs to carefully read by the developer to understand what's happening. Here these 2 lines are very obvious. I find it more readable the way I wrote it. Can you provide an argument about why it would be better the way you suggest?

Reply:
Better reading, simply. In the first time, I didn't see that it has two arguments.

Reply:
OK, I'll change it.

But your point does not work
- with arrays: you cannot use the number of lines to say "there's 8 lines so there must be 8 arguments"
- since all last items are empty strings, is that useful to know whether there's 3 or 4 empty strings?

I think it's easier to read my way, but I'll be open and change it. Anyway it's like apples or pears 🤷‍♂️  there's no right or wrong, just tastes.